Update war_in_heaven_events.txt

F1r3Pr1nc3 opened the pull request:
remove superfluous trigger war_in_heaven_seeking_league_leader
bit code opt

F1r3Pr1nc3 committed:
Update 00_bombardment_stances.txt

fix trigger scope has_claim

F1r3Pr1nc3 committed:
fix ending L-Gate event chain

https://forum.paradoxplaza.com/forum/threads/stellaris-3-0-3-event-suspected-cause-for-l-gate-event-chain-never-ending.1475690/

F1r3Pr1nc3 committed:
Fix L-Cluster outcome randomizing

https://forum.paradoxplaza.com/forum/threads/stellaris-l-cluster-outcome-spawning-only-spawning-l-drakes.1469825/

Ariphaos committed:
Merge branch 'master' into master

F1r3Pr1nc3 committed:
Update war_in_heaven_events.txt

remove superfluous trigger war_in_heaven_seeking_league_leader
bit code opt

Ariphaos commented on ChangeLog.md:
```diff
@@ -1,7 +1,9 @@
 # June 14th
 
-* Replaced one of Risa's fixes in on_actions to make it cleaner and more mod-friendly.
```
Uhh. Add your own date and fixes here, please.

F1r3Pr1nc3 committed:
Minor distar & marauder (& war_in_heaven)

- Minor performance fix for distar.10955 (monthly event) and some minor (old) syntax replacements (+ NOR/else_if /random_playable_country). (FirePrince)
- Minor fix for marauder.4: if a Marauder faction got destroyed by a non default empire an notify claims return but they do not. (FirePrince)
- Minor fix for marauder.6: add missing event_target:attacked_system. (FirePrince)

F1r3Pr1nc3 committed:
Minor fix bombardment stance raiding ai_weight scope.

F1r3Pr1nc3 committed:
Merge branch 'master' into master
